admin/acceptance.py

```diff
@@ -106,6 +106,13 @@ def start_nodes(self):
             cwd=self.vagrant_path.path,
             env=extend_environ(FLOCKER_BOX_VERSION=box_version))
 
+        from flocker.provision._install import run, task_pull_docker_images
+        for node in self.NODE_ADDRESSES:
+            run(
+                username='root',
+                address=node,
+                commands=task_pull_docker_images()
```

Comment:
`ACCEPTANCE_IMAGES` ought to be in `flocker.acceptance` somewhere, and therefore presumably not implicit in the provisioning code.